packs/classfeatures/weapon.json

```diff
@@ -28,6 +28,54 @@
             {
                 "key": "GrantItem",
                 "uuid": "Compendium.pf2e.actionspf2e.Item.Implement's Interruption"
+            },
+            {
+                "adjustName": false,
+                "choices": {
+                    "filter": [
+                        "item:level:0",
```

Comment:
What's the reason for the level 0 restriction?  The only limitation I see is that the starting implement must be a "mundane item of that type."  A level 1 weapon would seem to qualify.

There is of course the problem that player's will want to pick an item already in their inventory.

Reply:
Just an oversight. Going by non-magical is more accurate.